app/views/users/index.html.erb

```diff
@@ -9,6 +9,7 @@
           <th>Email</th>
           <th></th>
           <th></th>
+          <th></th>
```

Comment:
Oh haha yeah I don't mean it won't work, but isn't that breaking symantics? (As in explicit HTML)